omod/src/main/resources/apps/customLinks_app.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "id": "coreapps.customLinks",
+    "label": "coreapps.clinicianfacing.customLinks",
+    "icon": "icon-link",
+    "order": 10,
+    "extensions": [
+      {
+        "id": "${project.parent.groupId}.${project.parent.artifactId}.customLinks.clinicianDashboardSecondColumn",
+        "appId": "coreapps.customLinks",
+        "extensionPointId": "patientDashboard.secondColumnFragments",
+        "extensionParams": {
+          "provider": "${project.parent.artifactId}",
+          "fragment": "clinicianfacing/customLinksWidget"
+        }
+      }
+    ],
+    "config": {
+        "links": {
+          "patient dashboard": "/{{patientUuid}}",
+          "active visit": "/openmrs/coreapps/patientdashboard/patientDashboard.page?patientId={{patientUuid}}"
+        }
+    }
+  }
+]
```

Comment:
Could you actually remove this json file so that we do not see the section yet and document it at https://wiki.openmrs.org/display/docs/System+Administration%3A+Manage+Apps ?